ci-operator/step-registry/sandboxed-containers-operator/get-kata-rpm/sandboxed-containers-operator-get-kata-rpm-commands.sh

```diff
@@ -52,8 +52,21 @@ fi
 # calculate checksum
 KATA_RPM_MD5SUM=$(md5sum kata-containers.rpm | cut -d' ' -f1)
 
+# Upload and verify
+FAILED_NODES=""
 nodes=$(oc get node -l node-role.kubernetes.io/worker= -o name)
 for node in $nodes;do
     dd if=kata-containers.rpm| oc debug -n default -T "${node}" -- dd of=/host/var/local/kata-containers.rpm
-    oc debug -n default -T "${node}" -- bash -c "md5sum  /host/var/local/kata-containers.rpm.md5sum"
+    OUTPUT=$(oc debug -n default "${node}" -- sh -c "md5sum  /host/var/local/kata-containers.rpm")
```

Comment:
This seems a bit complex, wouldn't it be simpler to:

```bash
KATA_RPM_MD5SUM=$(cat kata-containers.rpm | md5sum)
...
dd if=kata-containers.rpm| oc debug -n default -T "${node}" -- dd of=/host/var/local/kata-containers.rpm
oc debug -n default "${node}" -- sh -c "echo $KATA_RPM_MD5SUM > md5sum /host/var/local/kata-containers.rpm.md5 && cat /host/var/local/kata-containers.rpm | md5sum -c /host/var/local/kata-containers.rpm.md5"
```

which should pass/fail based on the checksum?